config/session.js

```diff
@@ -12,4 +12,5 @@ module.exports = {
   proxy: true,
   resave: true,
   saveUninitialized: true,
+  rolling: true,
```

Comment:
I think this is fine, it just resets the expiration everytime the user does something, but this was required to get the flash messages to work (or at least test them) so the session/cookie exists to test the content.